File: 9.Palindrome Number.c
bool isPalindrome(int x) {
    if(x<0)
    {
        return false;
    }
    int i;
    int y=0;
    long int pal=0;
    int temp=x;
    while(x!=0)
    {
        y=x%10;
        x=x/10;
        pal=pal*10+y;
    }
    if(temp==pal){
        return true;
    }
    else{
        return false;
    }
}

File: 9. Palindrome Number.java
class Solution {
    public boolean isPalindrome(int x) {
        boolean result = true;
        if(x<10 && x>0){
            return result;
        }
        if(x<0){
            return false;
        }
        int realValue=x;
        int palin=0;
        while(x!=0)
        {
            int temp=x%10;
            palin=palin*10+temp;
            x/=10;
        }
        if(realValue!=palin){
            result=false;
        }
        return result;

    }
}
